var/spack/repos/builtin/packages/py-pyside2/package.py

```diff
@@ -0,0 +1,53 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyPyside2(PythonPackage):
+    """Python bindings for Qt."""
+    homepage = "https://pypi.org/project/PySide2/"
+    url      = "https://code.qt.io/pyside/pyside-setup.git"
+    git      = "https://code.qt.io/pyside/pyside-setup.git"
+
+    # More recent versions of PySide2 (for Qt5) have been taken under
+    # the offical Qt umbrella.  For more information, see:
+    # http://wiki.qt.io/Qt_for_Python_Development_Getting_Started
+
+    version('dev', tag='dev')
```

Comment:
```suggestion
    version('develop', tag='dev')
```
This will allow the versions to sort properly

Reply:
Will do.